### ISSUE TITLE
Wrong hreflang tag

### ISSUE THREAD

**Issue by dmetzler1988** [sulu/sulu#3174]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | maybe in all versions?
| Browser Version | everyone

#### Actual Behavior

If you have a language like `en-gb` then in the sulu meta tags the hreflang-tag will be shown as `en_gb` and this is wrong.

Full line:
```html
<link rel="alternate" href="http://demo.lo/en-gb" hreflang="en_gb"/>
```

#### Expected Behavior

the hreflang-tag should be correct and look like so: `en-gb`.

**Comment by danrot** [comment]
This is not as easy to solve correctly as initially thought... We would have to adapt the [`ParameterResolver`](https://github.com/sulu/sulu/blob/develop/src/Sulu/Bundle/WebsiteBundle/Resolver/ParameterResolver.php#L75), the [`DocumentInspector`](https://github.com/sulu/sulu/blob/develop/src/Sulu/Bundle/DocumentManagerBundle/Bridge/DocumentInspector.php#L317), because the locales would have match (could be avoided by doing the translations in the `ParameterResolver`).

Further the `locale` is also passed to the [`ContentPathTwigExtension`](https://github.com/sulu/sulu/blob/develop/src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php#L84), which does not match anymore afterwards. Two solutions for that: Either we do the string replace in the `seo.html.twig` file, or we also change the `ContentPathTwigExtension` (which would be quite a big BC break...).

I think I go with the simplest solution for now: doing a string replace for the `hreflang` attribute itself. Also keeps all the BC intact.

Sidenote: We have to get our localization representation right.

**Comment by danrot** [comment]
Fixed in #3177
